sdk/monitor/azure-monitor-ingestion/azure/monitor/ingestion/_operations/_patch.py

```diff
@@ -64,12 +64,12 @@ def upload(  # type: ignore[override] # pylint: disable=arguments-renamed, argum
                     content_encoding = "gzip"
                 logs.seek(0)
 
-            super().upload(rule_id, stream=stream_name, body=logs, content_encoding=content_encoding, **kwargs)
+            super()._upload(rule_id, stream=stream_name, body=logs, content_encoding=content_encoding, **kwargs)
```

Comment:
Do we still need the type: ignore on L29 or does it work now?

Reply:
Ooh, great catch! I can actually remove all the ignore markers on that line for both pylint and mypy.